README.md

```
@@ -1,7 +1,7 @@
 <div align="center">
   <img src="assets/images/ondine-logo.png" alt="Ondine Logo" width="600"/>
 
-  # Ondine - LLM Dataset Engine
+  # LLM Dataset Engine
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)